docs/scenes/get-started/set-up-your-environment/instructions/iosPhysicalDevelopmentBuild.mdx

```diff
@@ -0,0 +1,81 @@
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+
+import { BuildEnvironmentSwitch } from '../BuildEnvironmentSwitch';
+
+## Set up an iOS device with a development build
+
+<BuildEnvironmentSwitch type="xcode" />
+
+<Step label="1">
+### Enroll in the Apple Developer Program
+
+To install a development build on your iOS device, you will need an active subscription to the Apple Developer Program. You can sign up for the Apple Developer Program [here](https://developer.apple.com/programs/).
+
+</Step>
+
+<Step label="2">
+### Install EAS CLI
+
+To build your app, you will need to install EAS CLI. You can do this by running the following command in your terminal:
+
+<Terminal cmd={['$ npm install -g eas-cli']} />
+
+</Step>
+
+<Step label="3">
+
+### Create an Expo account and login
+
+Next, you will need to create an Expo account and login to the EAS CLI.
+
+1. [Sign up](https://expo.dev/signup) for an Expo account.
+2. Run the following command in your terminal to login to the EAS CLI:
+   <Terminal cmd={['$ eas login']} />
+
+</Step>
+
+<Step label="4">
+### Configure your project
+
+Run the following command to create an EAS config in your project:
+
+<Terminal cmd={['$ eas build:configure']} />
+
+</Step>
+
+<Step label="5">
+### Create an ad hoc provisioning profile
+
+To install a development build on your iOS device, you will need to create an ad hoc provisioning profile. Create one by running the following command in your terminal:
+
+<Terminal cmd={['$ eas device:create']} />
+
+</Step>
+
+<Step label="6">
+### Create a development build
+
+Run the following command to create a development build:
+
+<Terminal cmd={['$ eas build --platform ios --profile development']} />
+
+</Step>
+
+<Step label="7">
+### Turn on developer mode
+
+1. Open Settings > Privacy & Security, scroll down to the Developer Mode list item and navigate into it.
+2. Tap the switch to enable Developer Mode. After you do so, Settings presents an alert to warn you that Developer Mode reduces the security of your device. To continue enabling Developer Mode, tap the alert's Restart button.
+3. After the device restarts and you unlock it, the device shows an alert confirming that you want to enable Developer Mode. Tap Turn On, and enter your device passcode when prompted.
```

Comment:
```suggestion
1. Open "Settings" > "Privacy & Security", scroll down to the "Developer Mode" list item, and navigate into it.
2. Tap the switch to enable "Developer Mode". After you do so, Settings presents an alert to warn you that Developer Mode reduces your device's security. To continue enabling Developer Mode, tap the alert's "Restart" button.
3. After the device restarts and you unlock it, the device shows an alert confirming that you want to enable Developer Mode. Tap "Turn On", and enter your device passcode when prompted.
```

Let's highlight key setting name on the iOS device here (as we do below for scanning QR code in the camera and when the user has to tap on "Open in iTunes".